### ISSUE TITLE
UUID primary key support for mySQL.

### ISSUE THREAD

**Issue by kshenes** [prisma/prisma#4477]
**Describe the bug**
Prisma supports UUID to be tagged @id in data model v1.1, however, this does not work when the backend database in MySQL.

From the Docs:
The @id directive can be applied to scalar fields of types:

ID
UUID
Int

**To Reproduce**
standup prisma with a new MySQL database (See Getting Started)
https://www.prisma.io/docs/get-started/01-setting-up-prisma-new-database-JAVASCRIPT-a002/

Create a simple type with UUID tagged as @id 

```
type User {
  uuid: UUID! @id(strategy: AUTO)
  name: String!
}
```
Try to deploy using prisma deploy
User
x– The field uuid is marked as id must have one of the following types: ID!, Int!.

**Expected behavior**
that I would get a mySQL table with uuid as the PK (as the docs suggest)

**Versions (please complete the following information):**
 - Connector:  `MySQL`
 - Prisma Server: 1.32
 - `prisma` CLI: Prisma CLI version: prisma/1.32.0 (linux-x64) node-v10.15.3
 - OS: CentOS Linux release 7.6.1810 (Core)
 - other dependencies: `prisma-client`, `prisma-binding`, etc. if applicable
+-- prisma-client-lib@1.23.0-test.3
| +-- prisma-datamodel@1.23.0-alpha.1
| +-- prisma-generate-schema@1.23.0-alpha.173
| | `-- prisma-datamodel@1.23.0-alpha.173
graphql-yoga@1.16.7

**Additional context**

**Comment by pantharshit00** [comment]
@kshenes 

Also worth mentioning the by default we generate a CUID which is like UUID but more collision resistant, see: https://github.com/ericelliott/cuid#horizontal-scalability
Maybe you are fine using that in the meanwhile

**Comment by kshenes** [comment]
Yes, we are using the CUID as a primary key at the moment and adding a UUID field to all tables.  Our customer namespaces the UUID space so they can correlate origination events by UUID namespace.  I would be nice to have a single id.
